.github/ISSUE_TEMPLATE/--design-related.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "✨ Design-related"
+about: Capture design- and research-specific needs
+title: ''
+labels: design
+assignees: liaprins-czi
+
+---
+
+### Overview of design need
+- Is there an existing GitHub issue this design work pertains to? 
+- If so, provide a link to it
+- Also link to any specific comments or threads where the problem to be solved by design is mentioned
+- In a sentence or two, describe the problem to be solved for users.
+
+### What level of design is needed? (Choose all that apply)
+_This section may be updated by the designer / researcher working on this issue_
+
+- [ ] **Research:**  high-level recommendation/exploration of user needs, design heuristics, and / or best practices to inform a design experience  
+(Use this option when you feel there’s a challenge to be solved, but you’re curious about what the experience should be — may involve research studies to understand challenges/opportunities for design)
+- [ ] **User experience / conceptual:** organizing and structuring of information flow and content, including layout on screen or across multiple steps
+- [ ] **Visual:** creating mockups, icons, etc
+(If choosing this level alone, it means that the content to be mocked up and its organization is already known and specified)
+
+### Is design a blocker?
+- [ ] **Yes:** engineering cannot proceed without a design first
+- [ ] **No:** engineering can create a first version, and design can come in later to iterate and refine
+
+If selecting **Yes**, how much design input is needed to unblock engineering? For example, is a full, final visual design needed, or just a recommendation of which conceptual direction to go?
+
+### T-shirt size
+_To be completed by design / research only_
+
+- [ ] ●○○○○ XS: able to be done in a typical workday or two
+- [ ] ●●○○○ S: able to do within one sprint, if also working on other things in the sprint
+- [ ] ●●●○○ M: able to do within one sprint, if it is the sole focus
+- [ ] ●●●●○ L: will likely take more than one sprint to complete, but less than two
+- [ ] ●●●●● XL: a very large experience to be determined or overhauled, will need to be broken down further
```

Comment:
since the person submitting this doesn't choose these options, maybe it would be better to just create labels for these instead?

Reply:
Yes great point; it probably doesn't make sense as part of the template